src/qt/optionsmodel.cpp

```diff
@@ -562,6 +525,22 @@ bool OptionsModel::isRestartRequired()
     return settings.value("fRestartRequired", false).toBool();
 }
 
+void OptionsModel::setSST(double val)
+{
+    QSettings settings;
+    double oldVal = getSST();
+    if (oldVal != val) {
+        settings.setValue("SSTVal", val);
```

Comment:
we shouldn't be duplicating the stake split threshold storage, and only store it in the walletdb.

- I know that this is not from this PR and can be tackled later -

Reply:
done.